frontend/src/pages/index.tsx

```diff
@@ -17,21 +16,7 @@ const Home: NextPage = () => {
     <>
       <PageSEO title={t("page_title")} description={t("meta_description")} />
       <Hero />
-      <FullWidthAlert type="info" heading={t("alert_title")}>
-        <Trans
-          t={t}
-          i18nKey="alert"
-          components={{
-            LinkToGrants: (
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href={ExternalRoutes.GRANTS_HOME}
-              />
-            ),
-          }}
-        />
-      </FullWidthAlert>
+      <BetaAlert />
```

Comment:
It might even be better to put this in the `Layout` component so it's just on every page by default. But then `Layout` would need multiple `children` areas (or somethin'), because sometimes the alert appears in the middle of other content (e.g. the `Hero` is above the alert in /index).

Reply:
Suggest we leave this as-is for now.